cnxcommon/urlslug.py

```diff
@@ -32,7 +34,11 @@ def generate_slug(book_title, *other_titles):
         book_title = slugify(remove_html_tags(book_title))
         return book_title
```

Comment:
Should you also remove quotes from the book title?

Reply:
I'll leave that for later resolution as we aren't actually using the book slug ATM. It's not the intention of this change to DRY (Don't Repeat Yourself) things out. There is a way to write this function so it is DRY, which would correct the problem.